packages/terra-clinical-header/src/themes/orion-fusion-theme/Header.module.scss

```diff
@@ -0,0 +1,34 @@
+:local {
+  .orion-fusion-theme {
+    // .flex-header
+    --terra-clinical-header-align-items: center;
+    --terra-clinical-header-background-color: #f4f4f4;
+    --terra-clinical-header-border-bottom: 0.16666667rem solid #64696c;
```

Comment:
@jeremyfuksa you mentioned this style was correct, which is actually incorrect in terra-action-header: https://github.com/cerner/terra-core/pull/2714/files?file-filters%5B%5D=.md&file-filters%5B%5D=.png&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-88dce3fa259e93fb4378f904ec7774c0L5

Reply:
Yeah. Oops. Let's stick with `9b9fa1` as it is in action-header.

Reply:
Oops. I told you the wrong one. `2px solid #9b9fa1` was it.

Reply:
When comparing the fusion component & nav header, their value in the dom is `0.16666667rem solid #64696c` for the bottom border.

Reply:
Oh, yes. I see what you're saying. The border should match Fusion, so it looks like we need to update the theme in action header as well.